src/renderServer.ts

```diff
@@ -1,7 +1,9 @@
+// eslint-disable-next-line simple-import-sort/imports
```

Comment:
this messes up the sorting import, we require `@sentry/tracing` to be imported before `@sentry/profiling-node` so that we can patch over the lib